charts/sonarqube/templates/install-plugins.yaml

```diff
@@ -22,7 +22,7 @@ data:
       [ -e {{ .Values.sonarqubeFolder }}/extensions/downloads/* ] && rm {{ .Values.sonarqubeFolder }}/extensions/downloads/*
       {{ range $index, $val := .Values.plugins.install }}
       echo {{ $val | quote }} >> {{ $.Values.sonarqubeFolder }}/extensions/downloads/list{{ end }}
-      cat {{ .Values.sonarqubeFolder }}/extensions/downloads/list | xargs -n 1 -P 8 wget --directory-prefix {{ .Values.sonarqubeFolder }}/extensions/downloads --no-verbose
+      cat {{ .Values.sonarqubeFolder }}/extensions/downloads/list | xargs -n 1 -P 8 wget --directory-prefix {{ .Values.sonarqubeFolder }}/extensions/downloads --no-check-certificate --no-verbose
```

Comment:
I'd rather this be optional as if there are certificates, we'd want them to be valid.

Reply:
Thanks for this message. I do really understand and share this point of view.
I'll do my best to fit this request.